src/app/core/services/weatherAPI.service.js

```diff
@@ -7,12 +7,22 @@ export default function (app) {
 
         const key = weatherAPIconstants.APIkey;
 
-        this.getData = () => $http({
-                method: 'GET',
-                url: `http://api.weatherbit.io/v2.0/current?lat=48.4503701&lon=35.0779096&key=${key}`
-              })
-              .then(({ data }) => data)
-              .then(({ data }) => data)
-              .then(([ weatherData ]) => weatherData);
+        this.getForecast = (lat, long) => $http({
+          method: 'GET',
+          url: `http://api.weatherbit.io/v2.0/current?lat=${lat}&lon=${long}&key=${key}`
+        })
+        .then(({ data: { data } }) => data)
+        .then(([ weatherData ]) => {
```

Comment:
Because I have only one element in Array and this element is Object. Always. 
Response example:
`
{
    ...
    data: {
        ...
        data: [
            0: {...weatherData...}
        ]
        ...
    }
    ...
}
`